GCQS

simontrinh0911 opened the pull request:
ed25519_public_key
--

nanis94 commented:
comment

nanis94 commented:
chatch:master

nanis94 commented:
comment

ZatmanOptionz24 reviewed (approved):
diff --git a/README.md b/README.md
new file mode 100644
index 0000000..383ceb1
--- /dev/null
+++ b/README.md
@@ -0,0 +1,2 @@
+# Github
+pill muncher

ZatmanOptionz24 commented:
> ## ed25519_public_key

ZatmanOptionz24 commented:
From 045414f2ba60d05f76106a387fe57c752217a747 Mon Sep 17 00:00:00 2001
From: Chris Hatch <hatch@tuta.io>
Date: Wed, 10 Aug 2022 15:42:25 +0700
Subject: [PATCH] gitignore

---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index 05e7cdc27..14c2410e3 100644
--- a/.gitignore
+++ b/.gitignore
@@ -18,3 +18,4 @@ yarn-debug.log*
 yarn-error.log*
 yarn.lock
 
+.gitsigners

Majdian commented:
Hi

Hero228 commented:
ку

V1e2n3k4a5t6e7s8w9a10r11l12u13 commented:
[]()

mofad24 commented:
Ky

JAMESRINGO123 commented:
Be strong

za8457 reviewed (approved):
https://github.com/pi-apps/pi-explorer/blob/v1.4.0/src/languages/en.json

za8457 commented on .gitignore:
```diff
@@ -18,3 +18,4 @@ yarn-debug.log*
 yarn-error.log*
 yarn.lock
 
+.gitsigners
```
@ZatmanOptionz24

Hamikos1 replied:
Hello world 🌎

waheedanwa replied:
Hi everyone

shny10 replied:
Hello 🤟

Ile203 replied:
Hello from North Macedonia 🇲🇰

Chukzy1995 replied:
Hello sir help me

BelloShanwilu23 replied:
Hi

za8457 reviewed (approved):
0x095ba796bfe6f85f1ec9c6f851905893f1f38218

ZatmanOptionz24 commented:
How are you?

On Mon, Apr 17, 2023, 1:50 AM Mohammad Zaman Khan ***@***.***>
wrote:

> ***@***.**** approved this pull request.
>
> 0x095ba796bfe6f85f1ec9c6f851905893f1f38218
> ------------------------------
>
> In .gitignore
> <https://github.com/pi-apps/pi-explorer/pull/53#discussion_r1168071232>:
>
> > @@ -18,3 +18,4 @@ yarn-debug.log*
>  yarn-error.log*
>  yarn.lock
>
> +.gitsigners
>
> @ZatmanOptionz24 <https://github.com/ZatmanOptionz24>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pi-apps/pi-explorer/pull/53#pullrequestreview-1387118198>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A44WYIOPEVKTDDMGWSVXENLXBSHWRANCNFSM6AAAAAAR2W2OAE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Zia6674 reviewed (commented):
ok

Commit:
prepare for soroban testnet release:
- upgrade to latest stellar-sdk
- add testnet rpc address
- render footprint ops - bump and restore
- handle change to default export in stellar-sdk
- new fly deployment

Commit:
bump to 2.1.0

Commit:
another spot to skip NotFoundError's to sentry

Commit:
another spot to skip NotFoundError's to sentry

Commit:
fix contract id invalid handling
fix html in contract storage tab
add rendering of begin and end sponsorhip future reserves operations

Commit:
fix contract id invalid handling
fix html in contract storage tab
add rendering of begin and end sponsorhip future reserves operations

Commit:
render clawback operation

Commit:
render clawback operation

Commit:
render trustline_flags_updated effect

Commit:
render trustline_flags_updated effect

Commit:
increase number of concurrent connections

Commit:
add a sitemap.txt as i can see a bunch of robots looking for it (200+ 404s in 24hr)

Commit:
add a sitemap.txt as i can see a bunch of robots looking for it (200+ 404s in 24hr)

Commit:
update the soroban rpc link for testnet as it was incorrect

Commit:
add rendering of ops clawback_claimable_balance and set_trust_line_flags

Commit:
add effect claimable_balance_clawed_back

Commit:
added effects:
- liquidity_pool_deposited
- liquidity_pool_withdrew
added operations:
- LiquidityPoolDeposit
- LiquidityPoolWithdraw

Commit:
move download wasm file button to top part so it's visible for all tabs

Commit:
add interface tab to show Rust interface
add /interface endpoint to generate bindings in Rust
update Dockerfile to install Rust and the soroban cli

Commit:
merge back latest master changes - new ops

Commit:
restore remote api endpoint

Commit:
bump versions of api and app

Commit:
sync Dockerfile port with app port

Commit:
update port in fly.toml

Commit:
fix the wabt tool linking errors by upgrading the ubuntu docker base image
restore code tabs to using javascript highlighting instead of C

Commit:
directory updates - add assets from the top 10 by volume that were missing
remove some dead exchanges
setup new anchors

Commit:
Merge branch 'master' into soroban

Commit:
search fix from soroban branch

Commit:
fix case sensitive asset search (yUSDC now works)

Commit:
README.md npm run dev

V1e2n3k4a5t6e7s8w9a10r11l12u13 reviewed (commented):
[]()

jp-ryuji committed:
add liquidity pools listing view

Jeffbuleli commented:
Cool

8822433396 commented:
Tq


On Tue, 14 Nov, 2023, 6:28 pm Jeffbuleli, ***@***.***> wrote:

> Cool
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pi-apps/pi-explorer/pull/53#issuecomment-1810154892>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDWGHGBNFWJYCLF5QGAEF33YENTHHAVCNFSM6AAAAAAR2W2OAGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQMJQGE2TIOBZGI>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

Hamikos1 commented:
> Zdravo

Hello

jp-ryuji committed:
add npm modules for linter

jp-ryuji committed:
add husky and lint-staged config to package.json

jp-ryuji committed:
add @typescript-eslint/parser

jp-ryuji committed:
modify .eslintrc

jp-ryuji committed:
add .prettierrc

jp-ryuji committed:
add .husky/pre-commit

jp-ryuji committed:
add eslint-config-react-app

jp-ryuji committed:
fix lint:fix in package.json

jp-ryuji committed:
rm contradict eslint config

jp-ryuji committed:
rm contradict prettier config

jp-ryuji committed:
set tabWidth to 2

jp-ryuji committed:
rm "arrowParens": "always" as it is the default behavior

jp-ryuji committed:
Merge pull request #544 from chatch/setup-prettier

Setup prettier

jp-ryuji committed:
update readme on local development setting

jp-ryuji committed:
Merge pull request #548 from chatch/update-readme-on-local-dev-setting

update readme on local development setting

jp-ryuji committed:
apply linter with `npm run lint:fix`

Commit:
add eslint disable and comment to service-worker.js

Commit:
search.js: sdk not defined, import Keypair directly to fix

jp-ryuji committed:
fix formatter error in CustomNetworkButton.tsx

jp-ryuji committed:
Merge pull request #549 from chatch/apply-linter-v2

apply linter with `npm run lint:fix`

V1e2n3k4a5t6e7s8w9a10r11l12u13 commented:
Hello

jp-ryuji committed:
refactor liquidity pools listing view

jp-ryuji committed:
npm i -D jest ts-jest @types/jest

jp-ryuji committed:
npm i -D ts-node

jp-ryuji committed:
npx jest --init

jp-ryuji committed:
use ts-jest to transpile TypeScript

jp-ryuji committed:
npm i -D jest-environment-jsdom

jp-ryuji committed:
use jest-environment-jsdom instead of jsdom

jp-ryuji committed:
rename networks.test.js to networks.test.ts

jp-ryuji committed:
fix networks.test.ts

jp-ryuji committed:
Merge pull request #550 from chatch/setup-jest

Setup jest

Waheedanwar12 commented:
Me kyc issue please may kyc approved

On Fri, 2 Jun 2023, 10:03 am Andestra, ***@***.***> wrote:

> ***@***.**** approved this pull request.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pi-apps/pi-explorer/pull/53#pullrequestreview-1456650709>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7M2LUJBGPK3EHBIN4T65CDXJFXYNANCNFSM6AAAAAAR2W2OAE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

jp-ryuji committed:
add @remix-run/eslint-config to extends in .eslintrc

jp-ryuji committed:
disable lint error against RTL in file where RTL is not used

jp-ryuji committed:
npm run lint:fix

jp-ryuji committed:
rm unnecessary "type" from import statement

jp-ryuji committed:
npm i -D @swc/core @swc/jest

jp-ryuji committed:
add .swcrc

jp-ryuji committed:
npm i -D @testing-library/react @testing-library/jest-dom

jp-ryuji committed:
npm i -D jest-preview

aminghorbani1986 commented:
Yes I am

در تاریخ پنجشنبه ۲۳ نوامبر ۲۰۲۳،‏ ۲۰:۲۴ Waheedanwar12 <
***@***.***> نوشت:

> Me kyc issue please may kyc approved
>
> On Fri, 2 Jun 2023, 10:03 am Andestra, ***@***.***> wrote:
>
> > ***@***.**** approved this pull request.
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/pi-apps/pi-explorer/pull/53#pullrequestreview-1456650709>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/A7M2LUJBGPK3EHBIN4T65CDXJFXYNANCNFSM6AAAAAAR2W2OAE>
>
> > .
> > You are receiving this because you are subscribed to this thread.Message
> > ID: ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pi-apps/pi-explorer/pull/53#issuecomment-1824723701>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXE2TNTNVE2E45C5L2Y2DSLYF55VFAVCNFSM6AAAAAAR2W2OAGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQMRUG4ZDGNZQGE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

jp-ryuji committed:
modify jest.config.ts

jp-ryuji committed:
add jest.setup.ts

jp-ryuji committed:
import @testing-library/jest-dom in jest.setup.ts

jp-ryuji committed:
npm i -D npm-run-all

jp-ryuji committed:
add jest-preview and test:preview to npm scripts

jp-ryuji committed:
README.md npm test:preview

jp-ryuji committed:
disable collectCoverageFrom in jest.config.ts

jp-ryuji committed:
rm ts-jest and @types/jest

jp-ryuji committed:
npm i -D @testing-library/user-event

jp-ryuji committed:
npm i -D @types/jest

jp-ryuji committed:
Merge pull request #551 from chatch/use-remix-run-eslint-config

Use remix run eslint config

jp-ryuji committed:
Merge pull request #552 from chatch/setup-jest-preview-with-react-testing-library

Setup jest preview with react testing library

abas929 commented:
السلام عليكم

abas929 commented:
السلام عليكم

jp-ryuji committed:
npm init playwright@latest

jp-ryuji committed:
add playwright scripts to package.json

jp-ryuji committed:
mv tests-examples e2e-test-examples

jp-ryuji committed:
let jest ignore e2e test

jp-ryuji committed:
README.md run e2e test

jp-ryuji committed:
fix test commands

jp-ryuji committed:
disable playwright workflow on CI

jp-ryuji committed:
npm i -D @remix-run/testing

jp-ryuji committed:
Merge pull request #553 from chatch/setup-playwright

Setup playwright

jp-ryuji committed:
use CSS modules

jp-ryuji committed:
change default order to asc

jp-ryuji committed:
make asset url smaller

jp-ryuji committed:
add liquidity-pool(s) to ja.json

Commit:
merged in latest master which includes whitespace changes

Commit:
add missing import

jp-ryuji committed:
handle federated address lookup error

Commit:
#556 remove soroban-client dependency in favor of only stellar-sdk

jp-ryuji committed:
convert .eslintrc to .eslintrc.js

jp-ryuji committed:
rm example files provided by playwright

jp-ryuji committed:
shorten playwright timeout

jp-ryuji committed:
add e2e/production.spec.ts

jp-ryuji committed:
add playwright-production.yml

jp-ryuji committed:
check all pages in e2e/production.spec.ts

chatch committed:
Merge pull request #559 from chatch/506-handle-federated-address-lookup-error

handle federated address lookup error

chatch committed:
Merge pull request #560 from chatch/convert-eslintrc-to-js

convert .eslintrc to .eslintrc.js

jp-ryuji committed:
Merge branch 'master' into 540-add-liquidity-pool-listing-view

jp-ryuji committed:
Merge pull request #561 from chatch/run-playwright-against-production

Run playwright against production

jp-ryuji committed:
Merge pull request #557 from chatch/540-add-liquidity-pool-listing-view

add liquidity pool listing view

V1e2n3k4a5t6e7s8w9a10r11l12u13 commented:
[]()

chatch committed:
Merge pull request #533 from chatch/soroban

soroban development

Commit:
contract api has breaking changes again - update to work with these
bump stellar-sdk to 11.0.1 - stable release
bump stellarexplorer to 2.3.0 - soroban now in master

Commit:
small log warn change

Commit:
restore npm start script - required in deployed environment

Commit:
update fly.toml

jp-ryuji committed:
fulfill smoke test

Commit:
#564 start on restoring the custom network selector and add soroban RPC address

Commit:
upgrade to stellar-sdk 11.1.0

jp-ryuji committed:
add .vscode/settings.json

jp-ryuji committed:
toggle color themes

chatch committed:
Merge pull request #566 from chatch/fulfill-smoke-test

fulfill smoke test

chatch committed:
Merge pull request #567 from chatch/add-vscode-settings-json

add .vscode/settings.json

Commit:
fix a bunch of warnings coming out of eslint

Commit:
merge in latest master

Commit:
update server setup functions to look in the session

Commit:
workaround remix import issue: https://github.com/remix-run/remix/discussions/7666

upgraded remix v2 - latest 2.4.0 (as an attempt to fix the import issue but i've left this in as nice to be on the latest and appears to work well)

setup a cookie based session for storing custom network addresses

Commit:
saveAs: replace currently implementation with a simpler one and convert to typescript
more fixes from the remix v2 upgrade

jp-ryuji committed:
fix indent of package.json

jp-ryuji committed:
fix README

jp-ryuji committed:
npm i -D react-test-renderer @types/react-test-renderer jest-fetch-mock

jp-ryuji committed:
enable jest-fetch-mock

jp-ryuji committed:
rewrite test with react-testing-library

jp-ryuji committed:
enable running test on ci

jp-ryuji committed:
disable running test on ci

Commit:
add logging to capture case where op.value is undefined on Effects

Commit:
use 2 spaces in package-lock.json - resolves conflict with incoming PR that changes the formatting
add json to .eslintrc files and this makes the autoformat use 2 spaces in my env too

Commit:
ran audit fix to remove 10 vulnerabilities and hopefully bring closer to master version to resolve conflicts

Commit:
merge back main - just commit with conflicts - it's a nightmare - so will blow it away on main and create a fresh one

chatch committed:
Merge pull request #570 from chatch/restore-test-code

Restore test code

Commit:
regenerated package-lock.json after conlficts

Commit:
merge in resolve tests

Commit:
fix formatting to resolve conflicts

Commit:
merged back master

Commit:
update package-lock.json

chatch committed:
Merge pull request #565 from chatch/network-selector

restore custom network selector

Commit:
bump nodejs to active LTS 20 from 18

Commit:
restore npm test in the github action

Commit:
bump node to 20 in github actions as well

Commit:
update package-lock.json - @types/node was out of sync
update README.md - add status badge for e2e prod tests

Commit:
get custom network addresses in root and pass them down through the header in to the network selectors

jp-ryuji committed:
Merge branch 'master' into toggle-color-themes

jp-ryuji committed:
default to light theme and fix invisible parts

Commit:
fix import errors in server_request_utils.ts

Commit:
more warnings including ServerApi

jp-ryuji committed:
Merge branch 'master' into toggle-color-themes

Commit:
Merge branch 'master' into network-selector-pt2

Commit:
skip broken test for now

Commit:
for the 3 core network buttons, ensure custom addresses are cleared from the session by submitting a POST to /settings and having that endpoint do the redirect

Commit:
fixed clearing of session on click of core network selector buttons

chatch committed:
Merge pull request #572 from chatch/network-selector-pt2

custom network selector

Commit:
remove debug console.log's

18888651793 commented:
这是来自QQ邮箱的假期自动回复邮件。 您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

Hamikos1 reviewed (approved):
Apsolutno

Commit:
js-xdr -> @stellar/js-xdr

Hamikos1 commented on package-lock.json:
package-lock.json

aminghorbani1986 replied:
Hi

Abdulrahim011 replied:
Hey good day

Commit:
fix npm run start by using workaround suggested by a github issue in the remix repo
upgraded remix deps to 2.4.1

Commit:
#574 fix handling of ScVal void

Commit:
#577 change expected titles in production.spec.ts until 577 is fixed

khw5211314 commented:
你好



---Original---
From: "Amin Ghorbani ***@***.***&gt;
Date: Sun, Dec 31, 2023 05:30 AM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: Re: [pi-apps/pi-explorer] GCQS (PR #53)




 
@aminghorbani1986 commented on this pull request.
 
 
On package-lock.json:
 
Hi
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***&gt;

Commit:
#360 Link directly to alternative Horizon's (and Soroban RPC servers)

Commit:
2.4.0

V1e2n3k4a5t6e7s8w9a10r11l12u13 reviewed (commented):
[]()

Commit:
new logo

Commit:
merged back master

Sutherlandjoseph commented:
1

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: khw5211314 ***@***.***>
Sent: Saturday, December 30, 2023 7:56:10 PM
To: pi-apps/pi-explorer ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [pi-apps/pi-explorer] GCQS (PR #53)


你好



---Original---
From: "Amin Ghorbani ***@***.***&gt;
Date: Sun, Dec 31, 2023 05:30 AM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: Re: [pi-apps/pi-explorer] GCQS (PR #53)





@aminghorbani1986 commented on this pull request.


On package-lock.json:

Hi

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***&gt;

—
Reply to this email directly, view it on GitHub<https://github.com/pi-apps/pi-explorer/pull/53#issuecomment-1872638762>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AYIGK45SLFQTPRSNMT2TWCDYMCZ2VAVCNFSM6AAAAAAR2W2OAGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQNZSGYZTQNZWGI>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

jp-ryuji committed:
fix second arg of useEffect

jp-ryuji committed:
Merge pull request #578 from chatch/fix-second-arg-of-use-effect

fix second arg of useEffect

jp-ryuji committed:
fix open new window icon on light theme

jp-ryuji committed:
fix spec for BackendResourceBadgeButtonWithResourceModal

chatch committed:
Merge pull request #569 from chatch/toggle-color-themes

toggle color themes

jp-ryuji committed:
Revert "#577 change expected titles in production.spec.ts until 577 is fixed"

This reverts commit aef44545c569f243834d2d9112e09b13884ca668.

jp-ryuji committed:
update centralized exchanges

chatch committed:
Merge pull request #579 from chatch/revert-smoke-test

Revert smoke test

chatch committed:
Merge pull request #580 from chatch/update-centralized-exchanges

update centralized exchanges

jp-ryuji committed:
add liquidity pool details view

jp-ryuji committed:
enhance smoke test for liquidity pool details view

chatch committed:
Merge pull request #581 from chatch/liquidity-pool-details-view

Liquidity pool details view

Commit:
add services.ports section to fly.toml - is now mandatory

V1e2n3k4a5t6e7s8w9a10r11l12u13 reviewed (commented):
Ok

jp-ryuji committed:
npm install p-retry

jp-ryuji committed:
set module: commonjs to support dynamic import of p-retry

jp-ryuji committed:
add retry to server requests

jp-ryuji committed:
change retries of p-retry to 1 (2 attempts in total)

jp-ryuji committed:
refactor code using withRetry

jp-ryuji committed:
update asset page

chatch committed:
Merge pull request #583 from chatch/update-asset-page

update assets page

V1e2n3k4a5t6e7s8w9a10r11l12u13 reviewed (commented):
Ok

jp-ryuji committed:
Merge pull request #582 from chatch/add-retry-to-server-requests

Add retry to server requests

RamadanHash reviewed (commented):
👍

jp-ryuji committed:
claimable balance views

jp-ryuji committed:
add smoke test for claimable balances

jp-ryuji committed:
fix unit test for searchStrToPath

jp-ryuji committed:
update resources lists on readme

chatch committed:
Merge pull request #584 from chatch/claimable-balance-views

Claimable balance views

chatch committed:
Merge pull request #585 from chatch/update-resources-lists-on-readme

update resources lists on readme

Commit:
audit fix - removed one high severity vulnerability

Commit:
bump to 2.5.0 for release

Commit:
bump to 2.6.0-beta

Commit:
fix the assets e2e test

jp-ryuji committed:
fix pagination link

chatch committed:
Merge pull request #588 from chatch/fix-pagination-link

fix pagination link

V2FWH88QGH reviewed (approved):
GBP7Y6FESRGXPQWGIVA7SSVLRIVOKLFQHZZOBVF4QTRKWTRIBILCAINW

V2FWH88QGH commented on app/components/ClaimableBalanceTable.tsx:
GD2VTBFF56BPQJOLNO3DZSMLDJMGC4GB2RFHGJ7N5RJGRODTTZ5ZHTDY

V2FWH88QGH replied:
GD2VTBFF56BPQJOLNO3DZSMLDJMGC4GB2RFHGJ7N5RJGRODTTZ5ZHTDY

V2FWH88QGH commented on README.md:
```diff
@@ -54,58 +60,99 @@ Local: http://localhost:3000
 
 ## Exploring Private / Local Development Networks<a name="private-networks"></a>
 
-steexp will connect to a local horizon instance at http://localhost:8000 by default. If your running a local private network for development this is quite handy for browsing your changes to the ledger.
```
GD2VTBFF56BPQJOLNO3DZSMLDJMGC4GB2RFHGJ7N5RJGRODTTZ5ZHTDY

V2FWH88QGH replied:
3000

Abdulrahim011 replied:
Can I have the money right knaw

V1e2n3k4a5t6e7s8w9a10r11l12u13 replied:
Like

V2FWH88QGH reviewed (commented):
GCCAFWTTEVRJ5UJW76FQYW3VA3FW4O4RIR6WZQHCDJFOXKF45O4QHLOK

Boonteang commented:
GCCAFWTTEVRJ5UJW76FQYW3VA3FW4O4RIR6WZQHCDJFOXKF45O4QHLOK

Boonteang commented:
Ky

Abdulrahim011 commented:
I need to sell my pi

sanjibmodak commented:
Sent from Yahoo Mail on Android 
 
  On Mon, Feb 26, 2024 at 3:13 PM, ***@***.***> wrote:   


@V1e2n3k4a5t6e7s8w9a10r11l12u13 commented on this pull request.

In README.md:
>  
 ## Exploring Private / Local Development Networks<a name="private-networks"></a>
 
-steexp will connect to a local horizon instance at http://localhost:8000 by default. If your running a local private network for development this is quite handy for browsing your changes to the ledger.

Like

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

V1e2n3k4a5t6e7s8w9a10r11l12u13 commented:
Chatch authored

Murad321213 commented:
GBP7Y6FESRGXPQWGIVA7SSVLRIVOKLFQHZZOBVF4QTRKWTRIBILCAINW

Commit:
move horizon requests for account balances (front) and tx page to the client browsers to avoid rate limiting on the backend server
more pages to come however deploy these first as they are popular routes

Commit:
Merge branch 'master' of github.com:chatch/stellarexplorer

Commit:
changes accounts root and transactions route to call horizon from the client

Commit:
move home page route to calling horizon from the browser instead of the server

Commit:
move horizon requests from server to browser for all the remaining accounts tabs

Commit:
just log the url for 404s instead of the response

Commit:
use an express entry point server.js to enable rate limiting middleware

Commit:
trust proxy = true
log a full detail access log type line per request
set host 0.0.0.0 explicitly - should avoid the fly error

Commit:
upgrade github action libraries - seeing warnings about these versions

Commit:
fix playwright tests that are failing intermittently
fix playwright action to upload report from the correct folder

Commit:
bump sentry to latest

Commit:
add public soroban rpc url - gateway.fm

Davidoo23 commented:
Hello I am new here,I want to know how I can sell my pi

Jdjdjd1209 reviewed (approved):
I need $

SURAJpradhan123 commented:
Help me

fouretraders commented:
Pi The biggest experience of my life

siruschavoshi commented:
I don't have good information to do this kind of work, please help me

در تاریخ دوشنبه ۱۲ اوت ۲۰۲۴،‏ ۱۷:۵۱ shny10 ***@***.***> نوشت:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In .gitignore
> <https://github.com/pi-apps/pi-explorer/pull/53#discussion_r1713869628>:
>
> > @@ -18,3 +18,4 @@ yarn-debug.log*
>  yarn-error.log*
>  yarn.lock
>
> +.gitsigners
>
> Hello 🤟
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pi-apps/pi-explorer/pull/53#discussion_r1713869628>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUVM62GPNVXKFGDWNTGWXETZRDAHFAVCNFSM6AAAAAAR2W2OAGVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDEMZTGE4DEOBSGU>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

626252619127 commented:
npm i && npm start

florim1213 commented:
https://TONFree.win/?r=99933
On Thu, 5 Sep 2024 at 13:19, 626252619127 ***@***.***> wrote:

> npm i && npm start
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pi-apps/pi-explorer/pull/53#issuecomment-2331261792>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BD3OMLDTE5ABPGLZPIQITULZVA45XAVCNFSM6AAAAAAR2W2OAGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGMZRGI3DCNZZGI>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

Sanjai2019 commented:
> ## ed25519_public_key

Commented 10/09/2024

Anzac15 commented:
Hi


On Mon, 9 Sep 2024, 03:53 Sanjai2019, ***@***.***> wrote:

> ed25519_public_key
>
> Commented 10/09/2024
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pi-apps/pi-explorer/pull/53#issuecomment-2336804092>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXAXV4SOV2OOEIAF35DHK5TZVSTLPAVCNFSM6AAAAAAR2W2OAGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGMZWHAYDIMBZGI>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

florim1213 commented:
https://TONFree.win/?r=99933


On Sun, 8 Sep 2024 at 21:52, Sanjai2019 ***@***.***> wrote:

> ed25519_public_key
>
> Commented 10/09/2024
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pi-apps/pi-explorer/pull/53#issuecomment-2336804092>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BD3OMLCA5R5YBVKUT3WBUADZVSTH5AVCNFSM6AAAAAAR2W2OAGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGMZWHAYDIMBZGI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

florim1213 commented:
https://TONFree.win/?r=99933


On Mon, 9 Sep 2024 at 00:02, Anzac15 ***@***.***> wrote:

> Hi
>
>
> On Mon, 9 Sep 2024, 03:53 Sanjai2019, ***@***.***> wrote:
>
> > ed25519_public_key
> >
> > Commented 10/09/2024
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/pi-apps/pi-explorer/pull/53#issuecomment-2336804092>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AXAXV4SOV2OOEIAF35DHK5TZVSTLPAVCNFSM6AAAAAAR2W2OAGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGMZWHAYDIMBZGI>
>
> > .
> > You are receiving this because you are subscribed to this thread.Message
> > ID: ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pi-apps/pi-explorer/pull/53#issuecomment-2336838442>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BD3OMLGN42LHU3BPCGVZMTTZVTCRDAVCNFSM6AAAAAAR2W2OAGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGMZWHAZTQNBUGI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

ojichenel commented:
Like

Commit:
fix github action on ubuntu 22.04.05 as latest just upgraded to 24.04 and broke this action

Commit:
change action node.js to use ubuntu 22.04.05 as well for consistency

Commit:
previous change to runner ubuntu-22.04.05 - i think because the patch level is not supported
dropping the patch level to see if this runs

SURAJpradhan123 commented:
![Uploading Screenshot_20250521-035619.png…]()

chatch committed:
add comment on assets page failure to fix issue seen today

chatch committed:
switch to pnpm for security

chatch committed:
upgrade stellar-sdk@11.1.0 -> @stellar/stellar-sdk@14.3.2

chatch committed:
2.7.0

chatch committed:
package-lock.json update

chatch committed:
fixes from stellar-sdk upgrade in v2.7.0

chatch committed:
merge in master

chatch committed:
continue moving from npm to pnpm

chatch committed:
update github action to use pnpm

chatch committed:
github action: install pnpm before setup-node

chatch committed:
github action: pnpm v10 and remove non existant --if-present

chatch committed:
refactor: Migrate Remix to Vite, updating server setup and loader APIs.

chatch committed:
fixing issues from runs after migration

chatch committed:
massive amount of changes - lint fixes, attempts at getting app workig after major refactor

chatch committed:
upgrade baseline-browser-mapping

chatch committed:
fixed sentry error

chatch committed:
disable sentry in dev

chatch committed:
refactor: remove redundant JSON parsing from data fetches, update agent guidelines, and add IPFS development notes.

chatch committed:
Refactor navigation and network switching to leverage Remix components and direct URL updates, also remove LiveReload.

chatch committed:
feat: implement fallback mechanism for clipboard copy functionality

18888651793 commented:
这是来自QQ邮箱的假期自动回复邮件。 您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

chatch committed:
finished conversion to client only setup

chatch committed:
switch api/ to pnpm
add wrangler to ui package.json

chatch committed:
setup pages:dev and wrangler.toml

chatch committed:
node 24

chatch committed:
Merge pull request #703 from chatch/client_only_antigravity

client only

chatch committed:
3.0.0

chatch committed:
remove cloudflare insights script as we now run on cloudflare and get analytics built in

chatch committed:
update workflow to use pnpm

chatch committed:
install playwright browsers with pnpm and cache

chatch committed:
flip playwright tests back to steexp.com

chatch committed:
fix /effects

chatch committed:
3.0.1

chatch committed:
handle bot 404s and 405s streaming into sentry

chatch committed:
attempt to fix flaky test

chatch committed:
Filter Remix route miss Sentry noise

chatch committed:
Merge pull request #706 from chatch/sentry-filter-route-miss-noise

Filter Remix route miss Sentry noise

chatch committed:
remove old fly deployment for the app
describe deployments in AGENTS.md

chatch committed:
chore: configure pnpm 11 allowBuilds and bump wrangler to ^4.107.0

pnpm 11 introduced a stricter build-script approval gate; without an explicit
allow-list, `pnpm install` aborts on postinstall scripts in transitive
deps (esbuild, sharp, workerd, @swc/core, @sentry/cli, @stellar/stellar-sdk,
jest-preview, unrs-resolver). Add a pnpm-workspace.yaml with the canonical
v11+ `allowBuilds` map so local dev and `pnpm pages:deploy` work again
on a clean checkout.

Bump the wrangler devDependency from ^4.63.0 to ^4.107.0 (along with
the corresponding lockfile churn) — picked up during the same install
when v11 approval was being configured.

No application code changes; production deploy via `pnpm pages:deploy`
went out successfully immediately before this commit.

chatch committed:
Add GitHub security checks and update dependencies

chatch committed:
Fix pnpm setup in GitHub workflows

chatch committed:
Override vulnerable transitive dev dependencies

chatch committed:
Limit Dependabot version updates to non-major bumps

chatch committed:
Use execFile for uploaded wasm tooling

chatch committed:
Harden uploaded wasm path handling

chatch committed:
Avoid file metadata in wasm cleanup paths

chatch committed:
Tighten steexp trace target regex

chatch committed:
Avoid shell pipeline in exchange restructure script

chatch committed:
Add tag-triggered deploy workflow

chatch committed:
Merge pull request #712 from chatch/tag-deploy-actions

Add tag-triggered deploy workflow

chatch committed:
Bump version to 3.1.0

chatch committed:
Split app and API deploy tags

chatch committed:
Bump app version to 3.1.1

chatch committed:
Bump API version to 1.0.2

chatch committed:
Pin API Rust toolchain for soroban build

chatch committed:
Fix API Docker deploy

dependabot[bot] committed:
Bump ubuntu from 22.04 to 26.04 in /api

Bumps ubuntu from 22.04 to 26.04.

---
updated-dependencies:
- dependency-name: ubuntu
  dependency-version: '26.04'
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

chatch committed:
Fix production e2e Playwright version detection

chatch committed:
Stabilize production claimable balances e2e

dependabot[bot] committed:
Bump the github-actions group across 1 directory with 4 updates (#714)

Bumps the github-actions group with 4 updates in the / directory: [actions/checkout](https://github.com/actions/checkout), [actions/setup-node](https://github.com/actions/setup-node), [actions/cache](https://github.com/actions/cache) and [actions/upload-artifact](https://github.com/actions/upload-artifact).


Updates `actions/checkout` from 4 to 7
- [Release notes](https://github.com/actions/checkout/releases)
- [Commits](https://github.com/actions/checkout/compare/v4...v7)

Updates `actions/setup-node` from 4 to 6
- [Release notes](https://github.com/actions/setup-node/releases)
- [Commits](https://github.com/actions/setup-node/compare/v4...v6)

Updates `actions/cache` from 4 to 6
- [Release notes](https://github.com/actions/cache/releases)
- [Changelog](https://github.com/actions/cache/blob/main/RELEASES.md)
- [Commits](https://github.com/actions/cache/compare/v4...v6)

Updates `actions/upload-artifact` from 4 to 7
- [Release notes](https://github.com/actions/upload-artifact/releases)
- [Commits](https://github.com/actions/upload-artifact/compare/v4...v7)

---
updated-dependencies:
- dependency-name: actions/cache
  dependency-version: '6'
  dependency-type: direct:production
  update-type: version-update:semver-major
  dependency-group: github-actions
- dependency-name: actions/checkout
  dependency-version: '7'
  dependency-type: direct:production
  update-type: version-update:semver-major
  dependency-group: github-actions
- dependency-name: actions/setup-node
  dependency-version: '6'
  dependency-type: direct:production
  update-type: version-update:semver-major
  dependency-group: github-actions
- dependency-name: actions/upload-artifact
  dependency-version: '7'
  dependency-type: direct:production
  update-type: version-update:semver-major
  dependency-group: github-actions
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

chatch committed:
Upgrade and pin GitHub Actions (#715)

chatch committed:
Merge pull request #713 from chatch/dependabot/docker/api/ubuntu-26.04

Bump ubuntu from 22.04 to 26.04 in /api

chatch committed:
Integrate Sentry error reporting in API (v1.0.5)

- Add @sentry/node SDK; init at app startup reading SENTRY_DSN from env
- Capture exceptions in all CLI failure paths (stellar/wabt tooling)
- Wire Sentry.setupExpressErrorHandler for uncaught request errors
- DSN targets the new steexp-api project in the stellar-explorer org

chatch committed:
API: add test suite, central error handler, bump wabt to 1.0.37 (#717)

* Add API test suite, central error handler, and bump wabt to 1.0.37

- index.js: env-overridable tool paths/upload dir/max size; require.main
  listen guard + module.exports for testability; central error handler
  (multer LIMIT_FILE_SIZE -> 413, no stack leaks); wabt default 1.0.37.
  Sentry integration preserved intact.
- Dockerfile: pin wabt 1.0.37 (newest release with a prebuilt ubuntu
  binary; asset name gained -ubuntu-20.04 suffix).
- api/: standalone pnpm workspace root so its lockfile/store stay
  self-contained for the frozen-lockfile Docker build; jest + supertest
  devDeps; jest.config.js.
- api/__tests__/: 8 core cases (14 assertions). Setup auto-downloads wabt
  and generates a valid + truncated wasm that reproduces the recurring
  prod failures; /interface tested via a stellar shell stub.
- jest.config.ts (root): ignore api/ in the web-app suite.

* Fix CI: drop --ignore-workspace so allowBuilds is honored

The api job failed with ERR_PNPM_IGNORED_BUILDS for unrs-resolver@1.12.2.
The approval in api/pnpm-workspace.yaml (allowBuilds: unrs-resolver: true)
was correct, but --ignore-workspace makes pnpm skip pnpm-workspace.yaml
entirely, so the approval was never read and the install exited 1.

CI runs with working-directory: api, so pnpm already resolves to api's own
pnpm-workspace.yaml as the workspace root — no flag needed. Verified locally:
pnpm install --frozen-lockfile + pnpm audit --prod both exit 0.

Same latent bug existed in deploy.yml's deploy-api job (would have failed on
the next v*-api tag), fixed there too. Dockerfile (--prod, no flag) is unaffected.

* Run API tests in CI

The api job in node.js.yml only ran pnpm install and pnpm audit --prod,
leaving the jest suite (14 tests across 5 files in api/__tests__/) unrun.
Add pnpm test as a step after the audit.

The tests self-bootstrap via __tests__/setup/ensure-binaries.js, which
downloads wabt 1.0.37 with curl + tar and writes a stellar CLI stub, so
no extra setup is needed on the ubuntu-22.04 runner.

* Bump API version to 1.0.6

Bundles central error handler, wabt 1.0.37 bump, and the API test
suite/CI wiring since 1.0.5.

chatch committed:
Add access log for steexp-api (#718)

morgan middleware emits one line per finished request to stdout, which
Fly captures into `fly logs`. Format:

  2026-07-06T03:14:15Z POST /decompile 200 18432/51200 - 41.2 ms "curl/8.5.0" "https://steexp.com/contract/CABC…"

ISO timestamp, method, URL, status, response-bytes/request-bytes,
response time, user-agent, and referer. /health is skipped so a future
Fly health probe won't drown the log.

Includes three integration tests that pin the format, the error-path
logging, and the /health skip.

chatch committed:
Tag API access-log entries with contract ID (#719)

Browser default referrer-policy (`strict-origin-when-cross-origin`)
strips the URL path on cross-origin fetches, so the API's Referer
header was only seeing `https://steexp.com/` even though the call
originated from `/contract/<id>/code-wat`. We can't fix that from the
receiving side.

Fix: the web app now sends the contract ID it's already loading via
`X-Steexp-Contract-Id` on every fetch to /decompile, /wat, /interface.
The API logs it as a new field in the access log line (sanitised:
control chars stripped, capped at 100 chars) and emits `-` when
absent. Same grep-target ergonomics as before, just a more useful
field.

Web app: `postWasmToWabtBackendRoute(path)(wasmHex)` →
`(path)(contractId, wasmHex)`; same for getContractInterface.
Call sites updated in contract-code-tab-base.tsx and
contract..interface.tsx.

Bumps web app 3.1.1 → 3.1.2 and API 1.0.7 → 1.0.8.

chatch committed:
Add Paxos PYUSD to directory (searchable) + issuer logo

PYUSD was visible on the live /assets page (backend-driven) but not
searchable, since steexp's search box matches against the curated
directory. Add the Paxos anchor and PYUSD asset so searching "PYUSD"
resolves to /asset/PYUSD.

- anchor: token-metadata.paxos.com (Paxos, https://www.paxos.com/)
- asset:  PYUSD-GDQE7IXJ4HUHV6RQHIUPRJSEZE4DRS5WY577O2FY6YQ5LVWZ7JZTU2V5
  (regulated issuer; auth_revocable + clawback enabled), stablecoin
- logo:  PYUSDLOGO.png served at /img/token-metadata.paxos.com.png for
  the issuer account + curated /asset/PYUSD pages (<Logo> component)

Issuer home_domain, stellar.toml, and toml image all already point at
token-metadata.paxos.com, so the live /assets page was already correct.

chatch committed:
Redirect pool contract routes to contract pages

chatch committed:
Fix /asset/<code> crash: fetch live asset list + harden Asset row

Searching for an asset code (e.g. USD) redirected to /asset/USD, which
threw "Cannot access property 'length', s.trustlines is undefined".

Root cause: the Remix->Vite refactor (52651d0) wired the /asset/<id>
route to feed local directory records ({code,issuer,domain}) into the
Assets component, which expects the richer live-API shape
(supply/trustlines/payments/price7d/...). Any code with >1 directory
entry crashed.

- asset.$assetId.tsx: clientLoader now fetches matching assets from the
  same stellar.expert backend /assets uses (?search=<code>), returning a
  proper list of matching assets (the user's expectation). Guards
  non-JSON/error responses so it never crashes on a bad gateway.
- assets-base.tsx: harden the Asset row against records that lack
  price7d (common for low-activity assets in search results), trustlines,
  payments, supply, or domain. Missing price data renders an empty cell
  instead of throwing; missing domain hides the URL/server.toml cell.

Verified in-browser: /asset/USD renders 50 rows, /asset/PYUSD renders 7,
no errors. lint/typecheck/tests pass.

chatch committed:
Bump web app version to 3.1.3

Bundles the /asset/<code> crash fix, pool route redirects, and the PYUSD directory entry.
